docker/README.md

```diff
@@ -30,20 +39,53 @@ docker compose -f wildfly.yml -f mariadb.yml -f artemis.yml up
 
 FF!Test will be available at http://localhost/iaf-test
 
+## Debugging inside a Docker image
+
+It is possible to attach your Java debugger to a Docker image. You can place breakpoints but also live update classes to
+work on fixes easily, without the whole Maven build and Docker compose cycle.
+
+- First, build the basic image as described above with Maven, to build the test image.
+- Second, start the image with the following command, from the current 'docker' folder:
+  ```shell
+  docker compose -f tomcat-debug.yml up
+  ```
+  This is equivelent to:
+  ```shell
+  export VERSION=7.9-SNAPSHOT
+  docker run --rm \
+	-e JPDA_ADDRESS=8001 -e JPDA_TRANSPORT=dt_socket \
+	-p 80:8080 -p 8001:8001 \
+	-v ./../test/src/test/testtool:/opt/frank/testtool-ext \
+	-v ./../core/target/ibis-adapterframework-core-$VERSION.jar:/usr/local/tomcat/webapps/iaf-test/WEB-INF/lib/ibis-adapterframework-core-$VERSION.jar:ro \
+	ff-test:$VERSION-tomcat \
+	/usr/local/tomcat/bin/catalina.sh jpda run
+  ```
```

Comment:
Dit is alleen maar verwarrend en dan moet je straks dingen op 2 plekken onderhouden :)
```suggestion
```

Reply:
Je hebt een punt. Het hoeft niet dubbel. Dan houden we de compose file.